watchtower/wtdb/version.go

```diff
@@ -91,6 +97,27 @@ func getMigrations(versions []version, curVersion uint32) []version {
 	return updates
 }
 
+// GetLatestDBVersion returns the number of the latest database version
+// currently known to the wtclient.db.
+func GetLatestDBVersion(db kvdb.Backend) (uint32, error) {
```

Comment:
This name is a bit confusing, especially since it almost collides with `getLatestDBVersion` which returns something different (latest _known_ version vs. current database version).
So how about `CurrentDatabaseVersion()` instead?